client/hosting/hosting-features/components/hosting-features.tsx

```diff
@@ -61,32 +59,14 @@ const HostingFeatures = () => {
 
 	const hasEnTranslation = useHasEnTranslation();
 
-	const { data: siteTransferData, refetch: refetchSiteTransferData } = useSiteTransferStatusQuery(
-		siteId || undefined
-	);
-	// `siteTransferData?.isTransferring` is not a fully reliable indicator by itself, which is why
-	// we also look at `siteTransferData.status`
-	const isTransferInProgress =
+	const { data: siteTransferData } = useSiteTransferStatusQuery( siteId || undefined, {
+		refetchIntervalInBackground: true,
+	} );
+
+	const shouldRenderActivatingCopy =
 		( siteTransferData?.isTransferring || siteTransferData?.status === transferStates.COMPLETED ) &&
 		! isPlanExpired;
 
-	useEffect( () => {
-		if ( ! siteId ) {
-			return;
-		}
-
-		const interval = setInterval( () => {
-			if ( siteTransferData?.status !== transferStates.COMPLETED ) {
-				refetchSiteTransferData();
-			} else {
-				clearInterval( interval );
-				dispatch( fetchAtomicTransfer( siteId ) as unknown as AnyAction );
```

Comment:
This code refetched the Redux-stored site data when the Atomic transfer had been completed, right? Is it safe to remove it?

Reply:
Previously we needed this extra refetching as late as possible, since we didn't get actual information immediately, when a site becomes Atomic. Now we get everything at once in time, so we don't need extra refetching.

Reply:
I think I was misreading this before as a refetching of site details. I realize now that we were actually refetching data about the Atomic transfer.

I'm still curious, though: where are we consuming the Redux state about the Atomic transfer?